crates/proof-of-sql/src/base/scalar/mod.rs

```diff
@@ -3,18 +3,19 @@ mod scalar;
 pub use scalar::Scalar;
 mod error;
 pub use error::ScalarConversionError;
-mod mont_scalar;
-#[cfg(test)]
-mod mont_scalar_test;
-pub use mont_scalar::Curve25519Scalar;
 pub(crate) use mont_scalar::MontScalar;
 /// Module for a test Scalar
 #[cfg(test)]
 pub mod test_scalar;
 #[cfg(test)]
 mod test_scalar_test;
 
+/// TODO: Add docs
+pub mod mont_scalar;
+#[cfg(test)]
+mod mont_scalar_test;
```

Comment:
1) NIT: this doesn't need to get moved. It adds noise to the diff.
2) Why did `pub` get added? I could be wrong, but the `pub(crate) use mont_scalar::MontScalar;` re-export we already have should suffice.

Reply:
Hm, not sure if it was my auto formatting, I will recheck and revert as needed

Reply:
I was away from the computer so couldn't verify, I made this pub because if `test_scalar` is private, it's not possible to use from `record_batch_conversion.rs` tests. 

Made it `pub(crate)` instead of `pub` that way it stays inside crate. I should have done this first time.